force-app/main/default/aura/autoEnrollmentMappingModal/autoEnrollmentMappingModal.cmp

```diff
@@ -0,0 +1,17 @@
+<aura:component>
+    <aura:registerEvent name="modalDataChangeEvent" type="c:modalDataChangeEvent" />
+
+    <aura:attribute name="actionName" type="String" />
+    <aura:attribute name="accountRecordType" type="String" />
+    <aura:attribute name="autoProgramEnrollmentStatus" type="String" />
+    <aura:attribute name="autoProgramEnrollmentRole" type="String" />
+    abc{!v.autoProgramEnrollmentStatus}
+    <c:autoEnrollmentMappingModalBody
+        actionName="{!v.actionName}"
+        accountRecordType="{!v.accountRecordType}"
+        autoProgramEnrollmentStatus="{!v.autoProgramEnrollmentStatus}"
+        autoProgramEnrollmentRole="{!v.autoProgramEnrollmentRole}"
+        onaccountrecordtypechange="{!c.handleAccountRecordTypeChange}"
+        onautoenrollmentmappingstatuschange="{!c.handleAutoEnrollmentMappingStatusChange}"
```

Comment:
You'll need to add a role change event handler here too.

Reply:
yep i did, just havent pushed yet